opentelemetry-api/tests/metrics/test_measurement.py

```diff
@@ -0,0 +1,41 @@
+# Copyright The OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from unittest import TestCase
+
+from opentelemetry.metrics.measurement import Measurement
+
+
+class TestMeasurement(TestCase):
+    def test_measurement_init(self):
+        # int
+        Measurement(321, {"hello": "world"})
```

Comment:
Please follow [this](https://github.com/open-telemetry/opentelemetry-python/tree/metrics_new) approach when testing that no exception was raised.

Reply:
@ocelotl I think that is the wrong link, i don't follow

Reply:
Sorry, copy and paste is hard. [Here](https://github.com/open-telemetry/opentelemetry-python/blob/main/exporter/opentelemetry-exporter-otlp/tests/test_otlp.py#L24-L29) is the right link.